airflow/models/dagbag.py

```diff
@@ -467,17 +466,7 @@ def collect_dags(
             except Exception as e:  # pylint: disable=broad-except
                 self.log.exception(e)
 
-        end_dttm = timezone.utcnow()
-        durations = (end_dttm - start_dttm).total_seconds()
-        Stats.gauge('collect_dags', durations, 1)
-        Stats.gauge('dagbag_size', len(self.dags), 1)
```

Comment:
Should we remove it from https://github.com/apache/airflow/blob/master/docs/logging-monitoring/metrics.rst ?

Reply:
It's still emitted here https://github.com/apache/airflow/blob/0a1b434d2895c3245f02e33cec40a64a92777876/airflow/utils/dag_processing.py#L1142-L1151

Reply:
(It was probably a bug in my linked PR #6157 that this was still emitted here!)

Reply:
Hmmm curious, I _did_ remove this in that PR. How did it come back..

Oh. It got added back in in #5743. Whoops. So metric was fixed in 1.10.6 and then broken again in 1.10.7 :(